### DISCUSSION TITLE
GasUsage , fee calculation

### DISCUSSION THREAD

**Discussion by sankararaokgmail** [filecoin-project/lotus#6173] [General Q&A] [answered]
how to get the GasUsage , to calculate the burn fee? Is there a API to get this information?

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment, marked as answer]
no. you can get an estimate via api calls - but the final result is fixed when the message got included in a block.

https://docs.filecoin.io/build/get-started/ might be worth a look for how to access a nodes api
